### PR TITLE
[CARBONDATA-31] To unify definition for SparkContext,CarbonContext,HiveContext,SQLContext

### PR DESCRIPTION
**Pull request by chenliang613**
For SparkContext,CarbonContext,HiveContext,SQLContext definition, different places having different definition, for example:
1)val sc = new CaronContext, val carboncontext = new CaronContext
2)var oc: HiveContext , var context: HiveContext
....

To unify definition for SparkContext,CarbonContext,HiveContext,SQLContext as below:
CarbonContext : cc
SQLContext : sqlContext
SparkContext : sc
HiveContext : hiveContext

### CODE REVIEW

File: integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala [hunk 1]
```diff
@@ -17,8 +17,6 @@
```
Comment (ravipesala) [inline integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala]: It seems there is a problem in rebasing. It hasn't taken latest commits. This class not supposed to change.You overriding with old commit.

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala [hunk 2]
```diff
@@ -93,11 +93,11 @@ object CarbonDataRDDFactory extends Logging {
   }
 
   def mergeCarbonData(
-      sc: SQLContext,
-      carbonLoadModel: CarbonLoadModel,
-      storeLocation: String,
-      hdfsStoreLocation: String,
-      partitioner: Partitioner) {
+                       sqlContext: SQLContext,
```
Comment (ravipesala) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala]: Please don't change the format.

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/Compactor.scala [hunk 3]
```diff
@@ -41,14 +41,14 @@ object Compactor {
   val logger = LogServiceFactory.getLogService(Compactor.getClass.getName)
```
Comment (ravipesala) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/Compactor.scala]: Please don't change the format

File: integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala [hunk 4]
```diff
@@ -141,11 +138,7 @@ class CarbonStrategies(sqlContext: SQLContext) extends QueryPlanner[SparkPlan] {
         predicates,
         useUnsafeCoversion = false)(sqlContext)
       projectExprsNeedToDecode.addAll(scan.attributesNeedToDecode)
-      val updatedAttrs = scan.attributesRaw.map(attr =>
```
Comment (jackylk) [inline integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala]: why this code is deleted?
please create another PR if logic changed is required, and keep this PR change minimal

File: integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala [hunk 5]
```diff
@@ -251,13 +214,11 @@ class CarbonStrategies(sqlContext: SQLContext) extends QueryPlanner[SparkPlan] {
         }
       case d: HiveNativeCommand =>
         try {
-          val resolvedTable = sqlContext.executePlan(CarbonHiveSyntax.parse(d.sql)).optimizedPlan
+          val resolvedTable = sqlContext.executePlan(CarbonHiveSyntax.parse(d.sql)).analyzed
```
Comment (jackylk) [inline integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala]: why is it changed to analyzed?

File: integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala (hunk 4 above)
Comment (chenliang613) [inline integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala]: Let me check, these code doesn't need to delete.

**Comment by chenliang613** [comment]
close it ,create a new PR.

**Pull request closed without merging**